<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Entry</class>
 <widget class="QMainWindow" name="Entry">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>345</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>459</width>
    <height>345</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>459</width>
    <height>345</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #040F16;
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>459</width>
      <height>300</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabWidget {
	background-color: #040F16;
	color: #040F16;
	border: none;
}

QTabBar::tab {
  	background: #005E7C;
  	color: white;
  	padding: 10px;
}

QTabBar::tab:selected {
  	background: #001242;
	border-bottom-left-radius: 8px;
}</string>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>7</number>
    </property>
    <widget class="QWidget" name="Name">
     <property name="font">
      <font/>
     </property>
     <property name="accessibleName">
      <string/>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <attribute name="title">
      <string>Name</string>
     </attribute>
     <widget class="QLineEdit" name="first">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>27</y>
        <width>125</width>
        <height>47</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <italic>false</italic>
        <bold>true</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #005384;
border: 1px solid #5F8169;
border-radius: 0px;
color: white;
font: 600 23px &quot;Open Sans SemiBold&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>First</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="middle">
      <property name="geometry">
       <rect>
        <x>168</x>
        <y>27</y>
        <width>125</width>
        <height>47</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #005384;
border: 1px solid #5F8169;
border-radius: 0px;
color: white;
font: 600 23px &quot;Open Sans SemiBold&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Middle</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="last">
      <property name="geometry">
       <rect>
        <x>324</x>
        <y>27</y>
        <width>125</width>
        <height>47</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #005384;
border: 1px solid #5F8169;
border-radius: 0px;
color: white;
font: 600 23px &quot;Open Sans SemiBold&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Last</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="company">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>192</y>
        <width>439</width>
        <height>47</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <italic>false</italic>
        <bold>true</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="toolTip">
       <string/>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #005384;
border: 1px solid #5F8169;
border-radius: 0px;
color: white;
font: 600 23px &quot;Open Sans SemiBold&quot;;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Company</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="title">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>439</width>
        <height>47</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #005384;
border: 1px solid #5F8169;
border-radius: 0px;
color: white;
font: 600 23px &quot;Open Sans SemiBold&quot;;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Title</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Phones">
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <attribute name="title">
      <string>Phone</string>
     </attribute>
     <widget class="QLineEdit" name="home">
      <property name="geometry">
       <rect>
        <x>79</x>
        <y>27</y>
        <width>301</width>
        <height>47</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #005384;
border: 1px solid #5F8169;
border-radius: 0px;
color: white;
font: 600 21px &quot;Open Sans SemiBold&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Home</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="mobile">
      <property name="geometry">
       <rect>
        <x>79</x>
        <y>110</y>
        <width>301</width>
        <height>47</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #005384;
border: 1px solid #5F8169;
border-radius: 0px;
color: white;
font: 600 21px &quot;Open Sans SemiBold&quot;;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Mobile</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="work">
      <property name="geometry">
       <rect>
        <x>79</x>
        <y>192</y>
        <width>301</width>
        <height>47</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #005384;
border: 1px solid #5F8169;
border-radius: 0px;
color: white;
font: 600 21px &quot;Open Sans SemiBold&quot;;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Work</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Location">
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <attribute name="title">
      <string>Location</string>
     </attribute>
     <widget class="QLineEdit" name="address">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>57</y>
        <width>439</width>
        <height>47</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #005384;
border: 1px solid #5F8169;
border-radius: 0px;
color: white;
font: 600 23px &quot;Open Sans SemiBold&quot;;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Address</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="state">
      <property name="geometry">
       <rect>
        <x>168</x>
        <y>162</y>
        <width>125</width>
        <height>47</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #005384;
border: 1px solid #5F8169;
border-radius: 0px;
color: white;
font: 600 23px &quot;Open Sans SemiBold&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>State</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="city">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>162</y>
        <width>125</width>
        <height>47</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #005384;
border: 1px solid #5F8169;
border-radius: 0px;
color: white;
font: 600 23px &quot;Open Sans SemiBold&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>City</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="zip">
      <property name="geometry">
       <rect>
        <x>324</x>
        <y>162</y>
        <width>125</width>
        <height>47</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #005384;
border: 1px solid #5F8169;
border-radius: 0px;
color: white;
font: 600 23px &quot;Open Sans SemiBold&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Zip</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Online">
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <attribute name="title">
      <string>Online</string>
     </attribute>
     <widget class="QLineEdit" name="email">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>57</y>
        <width>439</width>
        <height>47</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #005384;
border: 1px solid #5F8169;
border-radius: 0px;
color: white;
font: 600 23px &quot;Open Sans SemiBold&quot;;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Email</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="website">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>162</y>
        <width>439</width>
        <height>47</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #005384;
border: 1px solid #5F8169;
border-radius: 0px;
color: white;
font: 600 23px &quot;Open Sans SemiBold&quot;;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Website</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Dates">
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <attribute name="title">
      <string>Dates</string>
     </attribute>
     <widget class="QLineEdit" name="birthday">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>57</y>
        <width>439</width>
        <height>47</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #005384;
border: 1px solid #5F8169;
border-radius: 0px;
color: white;
font: 600 23px &quot;Open Sans SemiBold&quot;;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Birthday</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="anniversary">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>162</y>
        <width>439</width>
        <height>47</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #005384;
border: 1px solid #5F8169;
border-radius: 0px;
color: white;
font: 600 23px &quot;Open Sans SemiBold&quot;;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Anniversary</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Notes">
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <attribute name="title">
      <string>Notes</string>
     </attribute>
     <widget class="QTextEdit" name="notes">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>439</width>
        <height>246</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #005384;
border: 1px solid #5F8169;
border-radius: 0px;
color: white;
font: 600 19px &quot;Open Sans SemiBold&quot;;</string>
      </property>
      <property name="placeholderText">
       <string>Notes</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Tags">
     <attribute name="title">
      <string>Tags</string>
     </attribute>
     <widget class="QListWidget" name="tags_list">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>439</width>
        <height>246</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QListWidget {
background: #005E7C;
font: 600 20px &quot;Open Sans SemiBold&quot;;
color: #040F16;
}
QListWidget:item:selected {
background: #005E7C;
color: white;
border: none;
}</string>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Image">
     <attribute name="title">
      <string>Image</string>
     </attribute>
     <widget class="QLabel" name="imageDisplay">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>55</y>
        <width>200</width>
        <height>200</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #005384;
border: 2px solid white;
border-radius: 0px;
color: white;
font: 600 19px &quot;Open Sans SemiBold&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="imageBtn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>439</width>
        <height>35</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background: #005384;
	border: 3px solid #005384;
	border-radius: 0px;
	font: 600 20px &quot;Open Sans SemiBold&quot;;
	color: white;
}

QPushButton:hover{
    background: #5F8169;
	border: 3px solid #1B2C2B;
}</string>
      </property>
      <property name="text">
       <string>Select Image</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="back">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>300</y>
      <width>50</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #0094C6;
border: none;
color: white;
border-top-right-radius: 8px;</string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="del">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>300</y>
      <width>50</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #040F16;
border: 3px solid crimson;
color: white;
border-top-right-radius: 8px;
border-top-left-radius: 8px;</string>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add">
    <property name="geometry">
     <rect>
      <x>409</x>
      <y>300</y>
      <width>50</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #0094C6;
border: none;
color: white;
border-top-left-radius: 8px;</string>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <zorder>tabWidget</zorder>
   <zorder>del</zorder>
   <zorder>back</zorder>
   <zorder>add</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
